Readme.md

```diff
@@ -1,22 +1,24 @@
-<!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128549718/14.1.3%2B)
-[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E20054)
-[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# GridView for MVC - How to add a column if the AutoGenerateColumns property is set to true
 
-* [HomeController.cs](./CS/AddColumnAtRunTime/Controllers/HomeController.cs) (VB: [HomeController.vb](./VB/AddColumnAtRunTime/Controllers/HomeController.vb))
-* [Data.cs](./CS/AddColumnAtRunTime/Models/Data.cs) (VB: [Data.vb](./VB/AddColumnAtRunTime/Models/Data.vb))
-* [Index.cshtml](./CS/AddColumnAtRunTime/Views/Home/Index.cshtml)
-* [GridPartial.cshtml](./CS/AddColumnAtRunTime/Views/Shared/GridPartial.cshtml)
-<!-- default file list end -->
-# GridView - How to add a column if the AutoGenerateColumns property is set to true
+When the [AutoGenerateColumns](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.AutoGenerateColumns) property is set to `true`, handle the [GridViewSettings.DataBound](http://docs.devexpress.devx/AspNetMvc/DevExpress.Web.Mvc.GridSettingsBase.DataBound) event to add a column to the GridView. Before adding, check whether a column already exists to avoid duplicate columns.
```

Comment:
```suggestion
When the [AutoGenerateColumns](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.AutoGenerateColumns) property is set to `true`, handle the [GridViewSettings.DataBound](http://docs.devexpress.devx/AspNetMvc/DevExpress.Web.Mvc.GridSettingsBase.DataBound) event to add a column to the GridView. Before adding the column, check whether a column already exists to avoid duplicate columns.
```